activerecord/test/cases/finder_test.rb

```diff
@@ -224,6 +225,14 @@ def test_exists_with_string
     assert_equal true, Subscriber.exists?("   ")
   end
 
+  def test_exists_with_strong_parameters
+    assert_equal false, Subscriber.exists?(nick: "foo")
```

Comment:
Am I missing something or this test is not using `Parameters`?

Reply:
🤦‍♂️ My bad, fixed now.